### PR TITLE
Add Picotable  `ChoiceFilter` defaults and only show active users and contacts by default (SHOOP-2236, SHOOP-2642)

### PR DESCRIPTION
**Pull request by shawnadelic**
Refs SHOOP-2236, SHOOP-2642, SHOOP-2687

### CODE REVIEW

**Comment by tulimaki** [comment]
I think there is a typo in third commit message.

File: shoop/admin/static_src/picotable/picotable.js [hunk 1]
```diff
@@ -228,6 +228,20 @@ const Picotable = (function(m, storage) {
 
     }
 
+    function getDefaultValues(ctrl) {
+        var data = ctrl.vm.data();
+
+        var filters = {};
+        for (var i=0; i < data.columns.length; i++){
+            var value = data.columns[i].filter.defaultChoice;
+            if (value){
```
Comment (tulimaki) [inline shoop/admin/static_src/picotable/picotable.js]: Missing whitespace

File: shoop/admin/static_src/picotable/picotable.js [hunk 2]
```diff
@@ -228,6 +228,20 @@ const Picotable = (function(m, storage) {
 
     }
 
+    function getDefaultValues(ctrl) {
+        var data = ctrl.vm.data();
```
Comment (tulimaki) [inline shoop/admin/static_src/picotable/picotable.js]: This should probably be const. Same goes with other variables in this function.

File: shoop/admin/static_src/picotable/picotable.js (hunk 1 above)
Comment (shawnadelic) [inline shoop/admin/static_src/picotable/picotable.js]: Do you mean between lines 234 and 235 or somewhere else?

File: shoop/admin/static_src/picotable/picotable.js (hunk 2 above)
Comment (shawnadelic) [inline shoop/admin/static_src/picotable/picotable.js]: What about line 236? Seems like that should stay var?

File: shoop/admin/static_src/picotable/picotable.js (hunk 1 above)
Comment (tulimaki) [inline shoop/admin/static_src/picotable/picotable.js]: I think it should be if (value) {

File: shoop/admin/static_src/picotable/picotable.js (hunk 2 above)
Comment (tulimaki) [inline shoop/admin/static_src/picotable/picotable.js]: maybe not in line 236

**Pull request merged**